changelog.d/10912.bugfix

```diff
@@ -0,0 +1 @@
+return E2E cross signing keys fetched over federation when to a client.
```

Comment:
This doesn't really make sense? Please can you expand it into something that will be meaningful to other synapse admins. 

Also, do we know when this bug was introduced? Please include that info.